src/base/preferences.cpp

```diff
@@ -896,6 +896,16 @@ void Preferences::resolvePeerCountries(const bool resolve)
     setValue("Preferences/Connection/ResolvePeerCountries", resolve);
 }
 
+QString Preferences::geoIPLicenseKey() const
+{
+    return value("Preferences/Connection/GeoIPLicenseKey").toString();
```

Comment:
I wouldn't use legacy configuration structure for new settings (it was bound to the Options dialog layout). IMO, it's better to have something like "GeoIP/LicenseKey".